playbooks/roce/roce.yaml

```diff
@@ -0,0 +1,143 @@
+---
+# Install Mellanox OFED
+- import_playbook: mofed-install.yaml
+  run_once: true
+  vars:
+    mofed_site_place: "MLNX_OFED-4.6-1.0.1.1"
+    mofed_file_name: "MLNX_OFED_LINUX-4.6-1.0.1.1-ubuntu16.04-x86_64.iso"
+  tags:
+    - mellanox_ofed
+
+# Activate VF for relevant PF
+- hosts: all
+  become: true
+  become_method: sudo
+  tasks:
+    - name: checking the Mellanox OFED is installed
+      stat:
+        path: /usr/bin/ofed_info
+      register: ofed_present
+    - include_tasks: vf-activate.yaml
+      with_items:
+        - "{{ pf1 }}"
+        - "{{ pf2 }}"
+        - "{{ pf3 }}"
+        - "{{ pf4 }}"
+      when: ofed_present.stat.exists
+  tags:
+    - vf_activation
+
+#Install Mellanox GPUDirect - WIP
+
+#Install python dependencies
+- hosts: kube-master
+  tasks:
+    - pip:
+        name: openshift
+    - name: check for module openshift
+      python_requirements_facts:
+        dependencies: 
+          - openshift
+  tags:
+    - dep_openshift
+
+#Download K8s SRIOV DP image
+- hosts: kube-node

```

Comment:
I would suggest adding the following here:

become: true
become_method: sudo

Without this, the "Pull docker image" tasks will fail when the deployment is run by a non-root user.

Reply:
Ok. I will update.